Real Analysis/real-analysis-notes.tex

```diff
@@ -142,4 +146,263 @@ \subsection*{(1/19/2017) Lecture 1}
+\begin{prop}
+  If $Y$ is a set and $\Ep \subset \P(Y)$, then there is a smallest (coarsest)
+topology $\Top(\Ep)$ on $Y$ which contains $\Ep$. 
+\end{prop}
+\begin{proof}
+  The proof of this fact is relatively straight forward and can be
+  found in almost any point-set topology reference.
+\end{proof}
+\begin{defn}
+  Let $(X,\Top)$ and $(X',\Top')$ be two topological spaces. A map $f:
+  X \to X'$ is called \emph{continuous} if and only if for all $U' \in
+  \Top'$, $f^{-1}(U') \in \Top$. 
+\end{defn}
+Given this definition, we can come up with a more concise description
+of the product topology as follows.
+\begin{defn}[Concise definition of product topology]
+  Given the same setup as the previous definition of product topology,
+  $\Top$ must be such that, for all $\alpha$ and $U_\alpha \in
+  \Top_\alpha$, \[
+    \pi_\alpha^{-1}(U_\alpha) \in \Top
+  \]
+  Thus, we say that the product topology is $\Top = \Top(\Ep)$
+  where \[
+    \Ep = \{\pi_{\alpha}^{-1}(U_\alpha) \mid \alpha \in A, U_\alpha
+    \in \Top_\alpha\}
+  \]
+\end{defn}
+\begin{rmk}
+  Let $(X_1,d_1), \ldots, (X_n,d_n)$ be a finite collection of metric
+  spaces. The product distance on $V = X_1 \times \cdots \times X_n$
+  is given by \[
+    d(x,y) = \min_{1 \leq i \leq n} d(x_i,y_i)
+  \]
+  Then, $\Top(d)$ is the product topology coming from
+  $(X_1,\Top(d_1)), \ldots, (X_n,\Top(d_n))$. The proof of this is an
+  exercise. 
+\end{rmk}
+Now, let us restrict ourselves to the countable product case. Let
+$(X_n,d_n)_{n \geq 1}$ be a sequence of metric spaces and
+$(X_n,\Top(d_n))$ be a topological space. Then, we can consider \[
+  X = \prod_{n=1}^\infty X_n
+\]
+with $\Top$ the product topology. Then, we want to find $d$ the
+distance on $X$ such that $\Top(d) = \Top$. The metric has to be such
+that, given any $x = (x_n)_{n \geq 1}$ and $y = (y_n)_{n \geq 1}$,
+$d(x,y)$ will actually converge. Thus, we let \[
+  d(x,y) = \sum_{n=1}^\infty 2^{-n} \min(1,d_n(x_n,y_n))
+\]
+\begin{lem}
+  If $(X,d)$ is a metric space, then \[
+    \tilde{d}(x,y) = \min(1,d(x,y))
+  \]
+  is also a metric and, moreover, they define the same topology.
+\end{lem}
+\begin{proof}
+  The proof of this lemma is relatively straight-forward, but
+  conceptually follows from the fact that topologies encode local
+  information, and locally these metrics agree. One also has to show
+  that this is actually a metric, but this also is not much work.
+\end{proof}
+\begin{prop}
+  Let $(X_n,d_n)_{n \geq 1}$ be a sequence of metric spaces. Then, $U
+  \in \Top$ is an open set in $X = \prod_{n=1}^\infty X_n$ if and only
+  if for all $x \in U$, there exist $k \geq 0$ and $i_1, < \cdots <
+  i_k \in \N$ and $r > 0$ such that \[
+    B = \{y \in X \mid d(x_{i_1},y_{i_1}) < \cdots <
+    d(x_{i_k},y_{i_k}) < r\} \subset U
+  \]
+\end{prop}
+\begin{proof}
+  For the proof, one notes that a $B$ of this form can also be written \[
+    B = \pi_{i_1}^{-1}(B(x_i,r)) \cap \cdots \cap \pi_{i_k}^{-1}(x_{i_k},r)
```

Comment:
and so the last thing should be the ball B(x_{i_k},r) or radius r, like the first thing.